api/Element.json

```
@@ -10497,6 +10497,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options_sanitizer_parameter": {
+          "__compat": {
+            "description": "`options.sanitizer` parameter",
+            "spec_url": "https://wicg.github.io/sanitizer-api/#dom-element-sethtmlunsafe",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "140"
```

Shouldn't this also be Nightly?
```suggestion
                "version_added": "preview"
```